selfservice/flow/verification/hook.go

```diff
@@ -0,0 +1,89 @@
+package verification
```

Comment:
This one needs a test :) You can probably copy it from one of the other flows more or less 1:1

Reply:
Done